common/post-fs-data.sh

```diff
@@ -14,11 +14,6 @@ patch_xml() {
   fi
   case $2 in
     *dialer_phenotype_flags*.xml)  VAR1=boolean; VAR2=string; VAR3=long;;
-    *mixer_paths*.xml) VAR1=ctl; VAR2=mixer;;
```

Comment:
I think this fucked up my bluetooth calling in the car.

Reply:
this wouldnt of done anything anyway so it cant be that. we werent modifying mixer_paths